tests/components/logbook/test_init.py

```diff
@@ -157,27 +160,51 @@ async def test_service_call_create_log_book_entry_no_message(hass_):
     assert len(calls) == 0
 
 
-def test_humanify_filter_sensor(hass_):
```

Comment:
This was not a good test, `humanify` should not be tested on its own.
In a follow-up we should rename `humanify` to `_humanify` to make it clear it's not to be used on its own.